hack/kind/Testing_Packages.md

```diff
@@ -0,0 +1,79 @@
+# Testing packages
```

Comment:
@anujc25 which of the 3 files in this directory are worth retaining?

Reply:
`deploy_kind_with_capi_and_kapp.sh` file depends on CAPx versions. If we are keeping this we might have to periodically update the versions there.
Other than that, I thing we can keep all 3 files here.